DCScribe/Core/Scribe.cs

```diff
@@ -54,25 +54,26 @@ public async Task ExecuteAsync(CancellationToken stoppingToken)
                 var scribeToken = scribeTokenSource.Token;
 
                 // Clear the database as we start from scratch each time around
-                await _databaseClient.ClearTableAsync();
-
-                /*
-                 * A queue containing all the unit updates to be processed. We populate
-                 * this queue in a separate thread to make sure that slowdowns in unit
-                 * processing do not impact the rate at which we can receive unit updates
-                 *
-                 * We clear the queue each time we connect
-                 */
-                var queue = new ConcurrentQueue<Unit>();
-                _rpcClient.UpdateQueue = queue;
+                await _databaseClient.ClearTableAsync();                
 
                 var tasks = new[]
                 {
-                    _rpcClient.StreamUnitsAsync(scribeToken), // Get the events and put them into the queue
-                    ProcessUnitQueue(queue, scribeToken), // Process the queue events into the units dictionary
                     ProcessAirbaseUpdates(scribeToken), // Process Airbase updates by calling an APi on a timer
                     ProcessMarkPanelUpdates(scribeToken) // Process MarkPanel updates by calling an API on a long timer
                 };
+                if (GameServer.UnitStream == true) {
```

Comment:
With the changes above this would change to something like (Assuming we have multiple things that care about the unit positions which we may well do for things like heatmap recording)

```csharp
if(GameServer.RecordUnitPositions.Enabled {
 // Blah
}
```

This will pave the way to allow refactoring on how the unit queue is processed (which I will do in a different PR as I have some use-cases in mind regarding heatmaps).

Reply:
Sounds good. I'll re-factor and add some commits soon.